content/en/docs/concepts/services-networking/_index.md

```diff
@@ -23,9 +21,14 @@ implementation (barring any intentional network segmentation policies):
    * agents on a node (e.g. system daemons, kubelet) can communicate with all
      pods on that node
 
-Note: For those platforms that support `Pods` running in the host network (e.g.
-Linux), when pods are attached to the host network of a node they can still communicate 
-with all pods on all nodes without NAT.
+{{< note >}}
+For those platforms that support `Pods` running in the host network (e.g.
+Linux):
+{{< /note >}}
+
+   * host-network pods of a node can connect directly with all pods IPs on all
```

Comment:
host-network pods of a node can connect directly to the  pod IP's on all

Reply:
done!